### PR TITLE
Make a README

### PR DESCRIPTION
**Pull request by Ravna**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# SecureDNA
+
+- You can find information about the project as a whole, including papers and a live demo, at the [main site](https://securedna.org).
```
Comment (lynn) [inline profile/README.md]:
```suggestion
# SecureDNA

SecureDNA is a free, non-profit screening platform designed to safeguard DNA synthesis everywhere.

- You can find information about the project as a whole, including papers and a live demo, at the [main site](https://securedna.org).
```

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+- Current precompiled `synthclient` binaries are available for [Ubuntu amd64](https://github.com/SecureDNA/ppa).
+- Source for the client and servers is available in our [monorepo](https://github.com/SecureDNA/SecureDNA).
+- The [quickdna]((https://github.com/SecureDNA/SecureDNA) is a general-purpose DNA manipulation tool but does not have to be separately downloaded to build or run the servers or client.
```
Comment (lynn) [inline profile/README.md]:
```suggestion
- [quickdna](https://github.com/SecureDNA/SecureDNA) is a general-purpose DNA manipulation tool but does not have to be separately downloaded to build or run the servers or client.
```

File: profile/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+- To use the production servers with real databases, you'll need to [obtain certificates](https://securedna.org/start/), but you can compile the entire system below and run test servers with test databases with your own certificates.
```
Comment (lynn) [inline profile/README.md]:
```suggestion
- To use the production servers with real databases, you'll need to [obtain certificates](https://securedna.org/start/), but you can compile the entire system below and run test servers with test databases and your own certificates.
```

File: profile/README.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# SecureDNA
+
+SecureDNA is a free, non-profit screening platform designed to safeguard DNA synthesis everywhere.
+
+- You can find information about the project as a whole, including papers and a live demo, at the [main site](https://securedna.org).
+- Current precompiled `synthclient` binaries are available for [Ubuntu amd64](https://github.com/SecureDNA/ppa).
+- Source for the client and servers is available in our [monorepo](https://github.com/SecureDNA/SecureDNA).
+- [quickdna](https://github.com/SecureDNA/SecureDNA) is a general-purpose DNA manipulation tool but does not have to be separately downloaded to build or run the servers or client.
```
Comment (swooster) [inline profile/README.md]:
```suggestion
- [quickdna](https://github.com/SecureDNA/quickdna) is a general-purpose DNA manipulation tool but does not have to be separately downloaded to build or run the servers or client.
```
(edited to add closing parenthesis to suggestion)

File: profile/README.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# SecureDNA
+
+SecureDNA is a free, non-profit screening platform designed to safeguard DNA synthesis everywhere.
+
+- You can find information about the project as a whole, including papers and a live demo, at the [main site](https://securedna.org).
+- Current precompiled `synthclient` binaries are available for [Ubuntu amd64](https://github.com/SecureDNA/ppa).
+- Source for the client and servers is available in our [monorepo](https://github.com/SecureDNA/SecureDNA).
```
Comment (swooster) [inline profile/README.md]: Total nitpick: For something official/public, "source code" sounds more proper to my ears.
```suggestion
- Source code for the client and servers is available in our [monorepo](https://github.com/SecureDNA/SecureDNA).
```

**Pull request merged**